packages/fast-jss-utilities/src/screen-reader.ts

```diff
@@ -0,0 +1,28 @@
+import * as CSS from "csstype";
+import { JSSUtilities } from "./typedef";
+
+export interface IScreenReaderOnly {
+    position: CSS.PositionProperty;
+    overflow: CSS.OverflowProperty;
+    clip: CSS.ClipProperty;
+    width: CSS.WidthProperty<string>;
+    height: CSS.HeightProperty<string>;
+    border: CSS.BorderProperty<string>;
+    padding: CSS.PaddingProperty<string>;
+    margin: CSS.MarginProperty<string>;
+}
+
+export type ScreenReaderOnly = JSSUtilities<IScreenReaderOnly>;
```

Comment:
What purpose is `JSSUtilities` fulfilling? Can you not just return `IScreenReaderOnly`?